src/meta-srv/src/service/procedure.rs

```diff
@@ -169,6 +172,78 @@ impl procedure_service_server::ProcedureService for Metasrv {
         Ok(Response::new(resp))
     }
 
+    async fn reconcile(&self, request: Request<ReconcileRequest>) -> GrpcResult<ReconcileResponse> {
+        if !self.is_leader() {
+            let resp = ReconcileResponse {
+                header: Some(ResponseHeader::failed(Error::is_not_leader())),
+                ..Default::default()
+            };
+            warn!(
+                "The current meta is not leader, but a `reconcile` request have reached the meta. Detail: {:?}.",
```

Comment:
```suggestion
                "The current metasrv is not leader, but a `reconcile` request have reached. Detail: {:?}.",
```

Reply:
Should update all the error msg in this file

Reply:
resolved in [02c98d6](https://github.com/GreptimeTeam/greptimedb/pull/6614/commits/02c98d689549241d0c8e52461ad54959345a2e08)